src/display/annotation_layer.js

```diff
@@ -1293,6 +1293,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
       element.disabled = this.data.readOnly;
       element.name = this.data.fieldName;
       element.tabIndex = 0;
+      element.title = this.data.dateFormat || this.data.timeFormat || "";
```

Comment:
Should we perhaps do this instead to avoid setting the title attribute at all if we don't have a good tooltip value? I'm not sure if an empty `title` property is not displayed at all or if it would result in a visual tooltip without any contents.

```
if (this.data.dateFormat || this.data.timeFormat) {
  element.title = this.data.dateFormat || this.data.timeFormat;
}
```